README-pt.md

```diff
@@ -38,13 +38,13 @@ Deseja levar seu aplicativo do Watson ao próximo nível? Quer aproveitar os ati
 ## 1. Clonar o repositório Clone o `watson-online-store` localmente.
 Em um terminal, execute `$ git clone https://github.com/ibm/watson-online-store`. Utilizaremos o arquivo [`data/workspace.json`](data/workspace.json) e a pasta [`data/ibm_store_html/`](data/ibm_store_html) 
 
-## 2. Criar serviços do Bluemix Crie os serviços a seguir: 
+## 2. Criar serviços do IBM Cloud Crie os serviços a seguir: 
 * [**Watson Conversation**](https://console.ng.bluemix.net/catalog/services/conversation) 
 * [**Watson Discovery**](https://console.ng.bluemix.net/catalog/services/discovery) 
 * [**Banco de dados NoSQL do Cloudant**](https://console.ng.bluemix.net/catalog/services/cloudant-nosql-db/) 
 
-## 3. Obter as credenciais de serviços do Bluemix e incluir no arquivo .env 
-Enquanto cria os serviços do Bluemix, você precisará criar credenciais de serviços e obter o nome do usuário e a senha:
+## 3. Obter as credenciais de serviços do IBM Cloud e incluir no arquivo .env 
```

Comment:
this would break the link from steps if it wasn't already broken by the translation